docs/howto/windows.md

```diff
@@ -0,0 +1,39 @@
+Small guide on how to use the terminal using WSL (Windows Subsystem for Linux) with the windows file system, and using it to perform any commands while running a react native android app in Android emulator on Windows.
+
+### Why use WSL?
+WSL is great for Windows users, as they essentially allow the use of Unix terminal while not having to dual boot their system or suffer the performance loss caused by running Linux in a virtual machine.
+
+### Install WSL
+Install your preferred distribution of Linux available in the Windows store (You must have Windows build 1607 or later to be able to do this). I used Ubuntu for this guide.
+
+### Accessing the filesystem
+You can access your Windows local drives by navigating to `/mnt/<drive-letter>`
+
+### Running npm commands
+First install node in your subsystem. For example, to install node 8.x in Ubuntu, run:
+
+```
+curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+sudo apt-get install -y nodejs
+```
+
+Install `react-native-cli` globally with `sudo npm i -g react-native-cli`.
+
+Zulip recommends using `yarn` over `npm` which comes with node.
+
+Install yarn with `sudo npm i -g yarn`
```

Comment:
We already have instructions for installing Yarn -- they say to go to https://yarnpkg.com/en/docs/install . No need to say something else. :-)